CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/MainNavigator.kt

```diff
@@ -0,0 +1,33 @@
+package co.nimblehq.coroutine.ui.screens
+
+import androidx.fragment.app.Fragment
+import co.nimblehq.coroutine.R
+import co.nimblehq.coroutine.ui.base.*
+import co.nimblehq.coroutine.ui.screens.home.HomeFragmentDirections.Companion.actionHomeFragmentToSecondFragment
+import co.nimblehq.coroutine.ui.screens.second.SecondBundle
+import javax.inject.Inject
+
+interface MainNavigator : BaseNavigator
```

Comment:
Seems this Navigator is using for Home Fragment navigation. Should it be `HomeNavigator` instead?